update_available_ios/lib/adapters/get_ios_package_version_impl.dart

```diff
@@ -12,10 +12,12 @@ GetIOSPackageVersion httpGetIOSPackageVersion() {
   return (String bundleId, {String? iosAppStoreRegion}) async {
     final Uri uri;
 
+    // Busting iTunes CDN cache
+    final String timeStamp = DateTime.now().millisecondsSinceEpoch.toString();
```

Comment:
```suggestion
    final timestamp = DateTime.now().millisecondsSinceEpoch;
```

* Timestamp is a single word, so the correct capitalization would be `timestamp`;
* This repository follows Effective Dart, which says "local variables shouldn't have type annotations and should be declared using just `var` or `final`";
* `toString()` is not necessary here, since string interpolation implicitly calls it.

Reply:
Change the other occurrences of `timeStamp` to `timestamp` too.